listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -85,3 +96,196 @@ def create_user_recording_recommendation_event(user_name):
     event_data['created'] = event_data['created'].timestamp()
     event_data['event_type'] = event_data['event_type'].value
     return jsonify(event_data)
+
+
+@user_timeline_event_api_bp.route('/user/<user_name>/feed/events', methods=['OPTIONS', 'GET'])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+def user_feed(user_name: str):
+    """ Get feed events for a user's timeline.
+
+    :param user_name: The MusicBrainz ID of the user whose timeline is being requested.
+    :type user_name: ``str``
+    :param max_ts: If you specify a ``max_ts`` timestamp, events with timestamps less than or equal to it will be returned
```

Comment:
Our get_listens endpoint uses non-inclusive timestamps and this endpoint uses inclusive-timestamps. Is there a compelling reason to use inclusive timestamps here? It would be nice to keep consistency across API calls.